app/views/readers/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_with url: "/readers", method: :post, local: true do |form| %><br>
+  <%= form.label :name %>
+  <%= form.text_field :name %><br><br>
+  <%= form.label :age %>
+  <%= form.number_field :age, in: 1.0..10.0, step: 1.0 %><br><br>
```

Comment:
add name and age label and field.  Age has range it must be in 2-100 and can only be incremented by one